site/reference/cli/fn/eval/README.md

```diff
@@ -75,6 +75,8 @@ fn-args:
 
 --image, i:
   Container image of the function to execute e.g. `gcr.io/kpt-fn/set-namespace:v0.1`.
+  For convenience, if full image path is not specified, `gcr.io/kpt-fn/` is added as default prefix.
+  e.g. instead of passing `gcr.io/kpt-fn/set-namespace:v0.1` you can pass `set-namespace:v0.1`.
```

Comment:
We need the same update for `render` docs as well.

Reply:
In kpt fn render reference docs, we don't mention about image field anywhere. So couldn't find a better place to add it. Instead, I have updated it everywhere in the book for brevity.